changelog.txt

```diff
@@ -7,6 +7,26 @@ OPENSEADRAGON CHANGELOG
 * Documentation fix (#1814 @kenanchristian)
 * Better cleanup on destruction, to avoid memory leaks (#1832 @JoFrMueller)
 * Miscellaneous code cleanup (#1840 @msalsbery)
+* Improved browser sniffing - detect EDGE and CHROMEEDGE browsers
```

Comment:
Feel free to officially bump up the in progress number at the top to 2.5.0, BTW :-)

Reply:
> We are actually putting the pull request number, not the issue number, on items in the change log, and including the username of the patch author. I know it may feel a little silly, but I think it's worth listing all of these changeling items separately like you're doing, but including the same PR number and author at the end of each one.

Changelog is next on my punch list since you mentioned the format in the last PR that landed  ;) ...fixing